utils/db.js

```diff
@@ -22,6 +22,14 @@ const sequelize = process.env.DATABASE_URL ?
             },
         });
 
+const User = sequelize.define('user',{
+    id : {
+         type      : Sequelize.INTEGER,
```

Comment:
I don't think there is any need. the access token is renewed everytime a user logs in.
We store userinfo and acess-token in the session. No need to persist it to db.